### PR TITLE
Pc 5001 home placeholders

### PR DESCRIPTION
**Pull request by Giregm**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-XXX

## Checklist

I have:

- [ ] Made sure the title of my PR follows the convention `[PC-XXX] <summary>`.
- [ ] Made sure my feature is working on the relevant real / virtual devices.
- [ ] Written **unit tests** for my feature.
- [ ] Added new reusable components to the AppComponents page and communicate to the team on slack
- [ ] Added a **screenshot** for UI tickets.

## Screenshots

| \*iOS - [Phone name] | \*Android - [Phone name] | Tablet - [Phone name] | Little - [Phone name] |
| -------------------- | :----------------------: | --------------------: | --------------------: |
| placeholder_PR_1 |                          |                       |                       |

## Deploy hard

If native code (ios/android) was modified, **after** the PR is merged, on the master branch, upgrade the **app version** (+1 patch):

- if you want an hard deployment of the testing environment, use `yarn version:testing` (this will create a commit with a tag)
- then run `git push --follow-tags`

### CODE REVIEW

File: src/features/home/components/HomeBodyPlaceholder.tsx [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+} from 'ui/theme'
+
+enum TileSize {
+  M = LENGTH_M,
+  L = LENGTH_L,
+}
+
+export const HomeBodyPlaceholder = () => (
+  <React.Fragment>
+    <OfferModulePlaceholder size={LENGTH_L} numberOfTiles={3} />
```
Comment (antoinewg) [inline src/features/home/components/HomeBodyPlaceholder.tsx:26]: tu peux en mettre un peu plus, pour iPad

Capture d’écran 2020-11-23 à 11 29 24

File: src/features/home/pages/Home.test.tsx [hunk 2]
```diff
@@ -22,81 +21,87 @@ jest.mock('features/auth/AuthContext', () => ({
...
+    jest.clearAllTimers()
+  })
+  it('should render placehodler loader for 3s', async () => {
+    const home = await homeRenderer(false, true)
+    expect(home.queryByTestId('HomeBodyPlaceholder-testID')).toBeTruthy()
+    jest.advanceTimersByTime(2900)
+    expect(home.queryByTestId('HomeBodyPlaceholder-testID')).toBeTruthy()
+    act(() => jest.advanceTimersByTime(100))
+    expect(home.queryByTestId('HomeBodyPlaceholder-testID')).toBeFalsy()
+  })
```
Comment (antoinewg) [inline src/features/home/pages/Home.test.tsx:37]: propre comme test 👍

**Review by antoinewg — COMMENTED** [review]
Nickel ! Juste une petite remarque pour les iPad

**Review by antoinewg — APPROVED** [review]
Nickel ! Juste une petite remarque pour les iPad

File: src/features/home/pages/Home.test.tsx (hunk 2 above)
Comment (Giregm) [reply src/features/home/pages/Home.test.tsx:37]: merci !

File: src/features/home/components/HomeBodyPlaceholder.tsx (hunk 1 above)
Comment (Giregm) [reply src/features/home/components/HomeBodyPlaceholder.tsx:26]: Je merge, et on ajoutera plus de placeholders ou non selon les retours UX

**Pull request merged**